routes/updateSchedule.js

```diff
@@ -0,0 +1,41 @@
+// Route that pdates the shows saved to current user's schedule
+
+var express = require('express');
+var router = express.Router();
+var User = require('../models/user.js');
+
+router.post('/updateSchedule', function(req, res){
+
+  // can't figure out why this variable is named 'savedshows[]'
+  var savedShows = req.body['savedShows[]'];
+  console.log("received " + savedShows.length + " shows");
+
+  // Get current user
+  User.find({"uid": req.session.id}).exec(function(err, docs){
+    if(err){
+      return res.send({errors:"Error finding profile. If issue persists, please message us on Facebook."})
+    }
+
+    currentUser = req.session.id;
+    console.log(currentUser);
+
+    // console.log("docs[0] is " + docs[0]);
+    // console.log("docs[0].savedToSchedule is " + docs[0].savedToSchedule);
+    // console.log("req.savedShows is " + req.savedShows);
+
+
+    // TODO: get saved shows from userPage.html
+    var savedToSchedule = [];
```

Comment:
what's the difference between this variable (`savedToSchedule`) and the `savedShows` variable you declared above?

Reply:
oops, this one was a mistake. i'll fix